### PR TITLE
AppbyTroye.KoodoReader version 1.4.0.811

### PR DESCRIPTION
**Pull request by VAllens**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44819)

### CODE REVIEW

File: manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: AppbyTroye.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AppbyTroye.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: AppbyTroye.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
Name        : Koodo Reader 1.4.0
Publisher   : App by Troye
Version     : 1.4.0
ProductCode : 233610fa-2bda-5a09-a37b-75e0bafa7920
Uninstall   : winget uninstall --id "233610fa-2bda-5a09-a37b-75e0bafa7920"
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: AppbyTroye.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (ItzLevvie) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0.811/AppbyTroye.KoodoReader.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```

**Comment by OfficialEsco** [comment]
Just noticed this pr was up a few days ago https://github.com/microsoft/winget-pkgs/pull/44424

**Comment by superusercode** [comment]
I can close mine if the data in this one is preferred

**Comment by OfficialEsco** [comment]
I would like to see a merge of the metadata since both are good

**Comment by superusercode** [comment]
looks like mine was automatically merged right now, so someone will just have to edit this metadata on top

**Comment by VAllens** [comment]
I renamed the version folder

**Comment by VAllens** [comment]
```yaml
...
PackageIdentifier: AppbyTroye.KoodoReader
PackageVersion: 1.4.0.811
PackageLocale: en-US
Publisher: App by Troye
PackageName: Koodo Reader
...
```

This information (field values) is automatically populated by the **`wingetcreate`** tool.
In particular, the `PackageIdentifier` field and the `PackageVersion` field, as well as the folder structure.

**It's worth noting: this information (field values) was not manually populated by me.**

@OfficialEsco @superusercode

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.installer.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+FileExtensions:
+- pdf
+- epub
+- mobi
+- azw3
+- fb2
+- cbr
+- cbz
+- cbt
+- djvu
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.installer.yaml:15]:
```suggestion
- djvu
ElevationRequirement: elevationRequired
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/troyeguo/koodo-reader/releases/download/v1.4.0/Koodo-Reader-1.4.0.exe
+  InstallerSha256: 797149FFD50C7CBD31DBC077D1CB0726F715AA5CD812EB6FC32713139C973A92
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.installer.yaml:22]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.en-US.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+Description: Koodo Reader is an all-in-one ebook reader that can help you better manage and study your ebooks. It's free and open-source.
+Moniker: KoodoReader
+Tags:
+- reader
+- ebook
+- ebook reader
+- pdf
+- epub
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.en-US.yaml:28]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.zh-CN.yaml [hunk 10]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.zh-CN.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.zh-CN.yaml [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+PackageName: Koodo Reader
+PackageUrl: https://github.com/troyeguo/koodo-reader/releases
+License: GNU Affero General Public License v3.0
+LicenseUrl: https://github.com/troyeguo/koodo-reader/blob/master/LICENSE
+Copyright: Copyright (c) 2020-2021 App by Troye
+CopyrightUrl: https://koodo.960960.xyz
+ShortDescription: 开源电子阅读解决方案
+Description: Koodo Reader 是一个开源免费的电子书阅读器，支持多达15种主流电子书格式， 内置笔记、高亮、翻译功能，助力高效书籍阅读和学习。
+ManifestType: locale
+ManifestVersion: 1.0.0
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.locale.zh-CN.yaml:21]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.yaml [hunk 12]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: AppbyTroye.KoodoReader
+PackageVersion: 1.4.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.yaml:8]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.yaml [hunk 13]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
```
Comment (OfficialEsco) [inline manifests/a/AppbyTroye/KoodoReader/1.4.0/AppbyTroye.KoodoReader.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
I kinda wanted Levvie to take this one, but sure
This one is missing `ManifestVersion: 1.1.0` and `ElevationRequirement: elevationRequired`

Ill remove `TroyeGuo.KoodoReader` while you fix those (not sure if my suggested changes will show up)

**Comment by VAllens** [comment]
Hi, @OfficialEsco 
I added scopes and installation switches.
Support install to program folder or current user folder according to scope.

**Review by OfficialEsco — APPROVED** [review]
> Hi, @OfficialEsco I added scopes and installation switches. Support install to program folder or current user folder according to scope.

Ahaaaa that might explain why it failed unattended without elevation... smooth

**Pull request merged**